src/main/java/com/openmrs/migrator/core/services/impl/BundledResourceLoader.java

```diff
@@ -0,0 +1,19 @@
+package com.openmrs.migrator.core.services.impl;
+
+import com.openmrs.migrator.core.services.ResourceLoader;
+import java.io.IOException;
+import java.io.InputStream;
+import org.springframework.stereotype.Component;
+
+/** Loads resources from the jar file */
+@Component
+public class BundledResourceLoader implements ResourceLoader {
```

Comment:
What do you think of moving these FileIO operations into a utilities class instead of as a service?

Reply:
I think we could do it. 
The utility class will have to be injected into `MergeService` which has a dependency on a component that needs only to know how to load a transformation from the filesystem.
Won't the utility class bring in extra functionality not needed in `MergeService`?

Reply:
I'm not sure I'm following...

The service also needs to be injected into the MergeService so either way there is a dependency. Which I think is fine - it will just be one class with all the FileIO specific functions which should be generic enough for anything to use it.

To be clear I don't there should be one utility class - better we have classes for each set of utilities that are related. So no need to worry about pulling in more than we need

Reply:
One class with all the FileIO specific functions sounds good. I'll move it to `FileIOUtilities`.